tests/functionaltests/Tests/UIKitTests/_UIGestureCoordinator.mm

```diff
@@ -0,0 +1,443 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
```

Comment:
actually - it should be UIGestureCoordinatorTests.mm

---
In reply to: [100645468](https://github.com/Microsoft/WinObjC/pull/1909#discussion_r100645468) [](ancestors = 100645468)